src/os/bluestore/BlueStore.cc

```diff
@@ -10601,6 +10619,8 @@ void BlueStore::_update_cache_logger()
   logger->set(l_bluestore_blobs, num_blobs);
   logger->set(l_bluestore_buffers, num_buffers);
   logger->set(l_bluestore_buffer_bytes, num_buffer_bytes);
+
+  logger->set(l_bluestore_omap_iterator_count, OmapIteratorImpl::get_instance_count());
```

Comment:
nit: you could modify l_bluestore_omap_iterator_count directly by inc/dec in constructor/destructor.
It is slower, but a bit cleaner I think.

Reply:
done